build.sbt

```diff
@@ -75,3 +70,15 @@ lazy val checkerPlugin = (projectMatrix in file("checker-plugin"))
   .settings(libraryDependencies ++= Seq(akkaTyped % Test, akkaPersistence % Test, akkaProjections % Test))
   .dependsOn(checkerLibrary)
   .jvmPlatform(scalaVersions = supportedScalaVersions)
+
+lazy val schemaDumpPlugin = (projectMatrix in file("sbt-dumpschema"))
+  .enablePlugins(SbtPlugin)
+  .settings(name := "sbt-dumpschema")
```

Comment:
Btw pls provide a line on how to refer to this plugin... preferably via jitpack since it's more self-contained (I can push Build.scala with such dependencies it to a repo used by other ppl) than relying on locally published packages.

Reply:
Actually @MarconZet how did your setup look like in the downstream project where you tried to refer to Jitpack-hosted sbt plugin? In particular, did you add a `resolver` in project/build.sbt rather than in build.sbt?

I can see there are some sbt-dumpschema artifacts on Jitpack: https://jitpack.io/com/github/VirtusLab/akka-safer-serializer/

Reply:
(bump)

Reply:
It's working now.
Later, I will add an instruction to the readme on how to use it.

Reply:
Great 🥳 I'd appreciate the instruction indeed